postgresql/config.go

```diff
@@ -248,7 +249,7 @@ func (c *Config) connParams() []string {
 }
 
 func (c *Config) connStr(database string) string {
-	host := c.Host
+	host = c.Host
```

Comment:
Would revert this change, it must now be overwriting some top-level `host` variable

Reply:
done